app/views/ansible_repository/_repository_form.html.haml

```diff
@@ -0,0 +1,97 @@
+%br
+%br
+.form-horizontal
+  %form#form_div{:name           => "angularForm",
+                 'ng-controller' => "repositoryFormController as vm",
+                 'ng-show'       => "vm.afterGet",
+                 'ng-cloak'      => '',
+                  "miq-form"     => true,
+                  "model"        => "vm.repositoryModel",
+                  "model-copy"   => 'vm.modelCopy',
+                 :novalidate     => true}
+    = render :partial => "layouts/flash_msg"
+    .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
+      %label.col-md-2.control-label
+        = _('Name')
+      .col-md-8
+        %input.form-control{:type          => "text",
+                              :name        => "name",
+                              "id"         => "name",
+                              'ng-model'   => "vm.repositoryModel.name",
+                              :maxlength   => MAX_NAME_LEN,
+                              :required    => "",
+                              :checkchange => true}
+        %span.help-block{"ng-show" => "angularForm.name.$error.required"}
+          = _("Required")
+    .form-group
+      %label.col-md-2.control-label
+        = _('Description')
+      .col-md-8
+        %input.form-control{:type        => "text",
+                            :name        => "description",
+                            "id"         => "description",
+                            'ng-model'   => "vm.repositoryModel.description",
+                            :checkchange => true}
+    .form-group
+      %label.col-md-2.control-label
+        = _('SCM type')
+      .col-md-8
+        = select_tag('scm_type',
+                     options_for_select([["#{_('GIT')}", "git"]], 'git'),
+                     "ng-model" => "vm.repositoryModel.scm_type",
+                     :disabled => true)
+    .form-group
```

Comment:
I think you need to add a class here to get validation message to show in red color
`.form-group{"ng-class" => "{'has-error': angularForm.scm_url.$invalid}"}`